client/components/mma/upgrade/UpgradeSupportStyles.ts

```diff
@@ -0,0 +1,48 @@
+import { css } from '@emotion/react';
+import { brand, headline, space, textSans } from '@guardian/source-foundations';
+
+export const sectionSpacing = css`
+	margin-top: ${space[4]}px;
+`;
+
+export const linkCss = css`
+	${textSans.medium()};
+	color: ${brand[400]};
+	font-weight: 700;
+	text-decoration-line: underline;
+	margin-top: 10px;
+	margin-left: 20px;
+	display: flex;
+	justify-content: center;
+`;
+
+export const headingCSS = css`
+	${headline.medium({ fontWeight: 'bold' })};
+	margin-top: 0;
+	margin-bottom: 0;
+`;
+
+export const paragraphCss = css`
+	margin-bottom: 32px;
+	font-size: 17px;
```

Comment:
this looks like it should be done with a `textSans` or some other source style instead?

Reply:
Good point - changed to `textSans.medium`